### DISCUSSION TITLE
How to fine tune orientation classifier?

### DISCUSSION THREAD

**Discussion by dvivarelli** [mindee/doctr#992] [Q&A]
Hello,
Sometime the recognition stage give me the wrong output because the orientation is wrong, so i would like to fine tune the model.
in reference folder i have seen the train script but it seems about character classification, not for orientation.

Is there any script available for that?

**Comment by frgfm** [comment]
Hi @dvivarelli :wave:

You are correct! So far we haven't provided the training script, which is highly similar to character classification. We'll have to clean that and share it on Github :)

@charlesmindee can you confirm that this was trained on rotated characters and not rotated words? :pray: 
I can draft the training script according to your answer!

**Reply by dvivarelli** [reply]
Hello,
any news about the training script?

**Reply by frgfm** [reply]
I just opened an issue #1025 to track this (staged for release 0.6.0) :ok_hand:
